pom.xml

```diff
@@ -367,7 +367,7 @@
         <plugin>
           <groupId>com.github.siom79.japicmp</groupId>
           <artifactId>japicmp-maven-plugin</artifactId>
-          <version>0.7.0</version>
+          <version>0.7.2</version>
```

Comment:
Upgrade to 0.9.0? There are few bug fixes in 0.8.x and 0.9.0 that I needed in netlet.